edb/server/protocol/auth_ext/http.py

```diff
@@ -192,8 +187,7 @@ async def handle_request(
             _fail_with_error(
                 response=response,
                 status=http.HTTPStatus.INTERNAL_SERVER_ERROR,
-                message=str(ex),
-                ex_type=edb_errors.InternalServerError,
+                ex=ex,
```

Comment:
Do we need to wrap this catch-all error in an `InternalServerError`, so it gets serialised with that type name?

Reply:
I'm not sure, to be honest. I guess that would be consistent though, so if that makes sense to you, I can do that!

Reply:
@jaclarke ⚡ [`892b5ba` (#6751)](https://github.com/edgedb/edgedb/pull/6751/commits/892b5ba759f83f79bc974b52c2be64967a45091c)